src/app/login/githublogin.tsx

```diff
@@ -32,11 +47,21 @@ const GithubLogin: React.FC = () => {
                 icon={<GithubIcon />}
                 iconPosition="left"
                 size="lg"
-                style={{ backgroundColor: 'black' }}
+                style={{ backgroundColor: 'black', marginRight: '20px' }}
                 onClick={handleGitHubLogin}
               >
                 Sign in with GitHub
               </Button>
+              <Button
+                variant="primary"
+                icon={<CustomLogoIcon />}
+                iconPosition="left"
+                size="lg"
+                style={{ backgroundColor: 'black' }}
+                onClick={handleSignUpClick}
+              >
+                Sign up
```

Comment:
```suggestion
                Join the beta (required)
```

Reply:
This wording isn't perfect, but indicates to the user that this is still in "beta" and that clicking this button is also "required" (once) until we improve this flow in the future.